src/calculateRentalCost.js

```diff
@@ -29,6 +29,16 @@
 
 function calculateRentalCost(days) {
   // write code here
+
+  let sumInDays = days * 40;
+
+  if (days >= 7) {
+    sumInDays -= 50;
```

Comment:
create variables for 50, 40, 20
<img width="820" alt="Screenshot 2023-04-01 at 19 03 57" src="https://user-images.githubusercontent.com/32853034/229304613-2058e593-cf91-409b-93e5-a47cf6fed66b.png">